### ISSUE TITLE
Ruffle.exe complains about OpenGL with --graphics dx12

### ISSUE THREAD

**Issue by divinity76** [ruffle-rs/ruffle#12139]
### Describe the bug

I got a win7 virtual machine running on VMWare running on Windows 10 as a hypervisor, and the desktop Ruffle.exe refuse to run on the VM, but the interesting part is that when ruffle.exe is invoked with `--graphics dx12` it still complains about OpenGL, not DirectX 🤔 which suggest something is wrong here, is the --graphics argument being ignored? look at this
```
ruffle.exe

Ruffle does not support OpenGL on Windows.

ruffle.exe --graphics dx12

Ruffle does not support OpenGL on Windows.

ruffle.exe --graphics vulkan

Ruffle does not support OpenGL on Windows.

ruffle.exe --graphics gl

Ruffle does not support OpenGL on Windows.
```

### Expected behavior

not giving an OpenGL error when invoked with `--graphics dx12`

### Affected platform

Desktop app

### Operating system

Windows 7 SP1 x64

### Additional information

observed on Ruffle 0.1.0-nightly (a35ea3d9f21706237068a4978bf5044798591656 2023-07-17)

**Comment by ActionWavele** [comment]
your first mistake was running ruffle on a virtual environment

**Comment by divinity76** [comment]
@ActionWavele nah, funfact, VMWare (but not VirtualBox) actually has good GPU support, there are several games you can run in a comfy ~20fps on VMWare, and comparatively ~1fps or outright crash on VirtaulBox. I suspect the problem is that Win7 only support DirectX <=11, while Ruffle needs DirectX >=12

- Adobe's Flash Projector works fine (with graphics acceleration) on the same VM

**Comment by ActionWavele** [comment]
if you insist, though I think VMs are not even considered when they develop this emulator, so you may run into issues like the rest of VM users

**Comment by n0samu** [comment]
An easy way to reproduce this on a typical Windows system is to run Ruffle with `-g metal`. That will also warn about OpenGL.
